src/wallet/spend.cpp

```diff
@@ -378,16 +378,14 @@ bool AttemptSelection(const CWallet& wallet, const CAmount& nTargetValue, const
 {
     setCoinsRet.clear();
     nValueRet = 0;
-    // Vector of results for use with waste calculation
-    // In order: calculated waste, selected inputs, selected input value (sum of input values)
-    // TODO: Use a struct representing the selection result
-    std::vector<std::tuple<CAmount, std::set<CInputCoin>, CAmount>> results;
+    // Vector of result and waste pairs. We will choose the best one based on waste.
```

Comment:
0211023c256dd5fde9ff82e832305c8b3c981d89:

Really nice that it's just `SelectionResults` instead of tuples now :D but the comment needs to be updated

Reply:
Done